packages/react-storybook-addon/src/hooks.ts

```diff
@@ -18,3 +19,7 @@ export interface FluentGlobals extends StorybookArgs {
 export function useGlobals(): [FluentGlobals, (newGlobals: FluentGlobals) => void] {
   return useStorybookGlobals();
 }
+
+export function setGlobalTheme(themeId: string): void {
```

Comment:
I thought that exporting this was clearner than using the storybook channels and messages directly in `react-components`. Open to push back for this